pallets/confidential-asset/src/lib.rs

```diff
@@ -76,48 +71,35 @@ pub struct MercatAccount {
     pub encryption_pub_key: EncryptionPubKey,
 }
 
+type Identity<T> = identity::Module<T>;
+
 decl_storage! {
     trait Store for Module<T: Trait> as ConfidentialAsset {
 
         /// Contains the mercat accounts for an identity.
         pub MercatAccounts get(fn mercat_account):
```

Comment:
I wonder if there is a typo. Should the fn name be `mercat_accounts`?

Reply:
Good catch.